streaming/java/streaming-state/src/main/java/org/ray/streaming/state/backend/StorageMode.java

```diff
@@ -0,0 +1,29 @@
+package org.ray.streaming.state.backend;
+
+/**
+ * Storage model.
```

Comment:
doc: linked to its usages. I cannot get what `StorageMode` used for within itself. "Storage" is quite general.

Reply:
StateStrategyEnum is more straightforward.